test-cases/scale/scale-20-200-20-cluster-resize.yaml

```diff
@@ -0,0 +1,34 @@
+test_duration: 6000
+
+keyspace_num: 0
+user_profile_table_count: 0
+add_cs_user_profiles_extra_tables: true
+
+n_loaders: 0
+n_db_nodes: 20
+add_node_cnt: 1
+cluster_target_size: 200
+
+instance_type_db: 'i4i.2xlarge'
+instance_type_monitor: 'm6i.xlarge'
+root_disk_size_monitor: 4000
+
+nemesis_class_name: 'NoOpMonkey'
+
+# This is in order to start the basic cluster faster
+use_legacy_cluster_init: false
+parallel_node_operations: true
+seeds_num: 2
+# Takes too long on big clusters
+cluster_health_check: false
+
+backtrace_decoding: false
+
+append_scylla_yaml:
+  enable_repair_based_node_ops: true
+
+run_fullscan: []
+
+simulated_racks: 0
```

Comment:
I do not think that the pipeline that uses this config, is running real AZs so we should use simulated racks